### ISSUE TITLE
repozo permissions error on production - 2

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#394]
Noted in this morning's log watch emails:

```
INFO: Please wait while backing up database file: /usr/local/plone-4.3/primary/var/filestorage/Data.fs to /usr/local/plone-4.3/primary/var/backups
Traceback (most recent call last):
  File "/usr/local/plone-4.3/primary/bin/repozo", line 332, in <module>
    import ZODB.scripts.repozo
  File "/usr/local/plone-4.3/primary/eggs/ZODB3-3.10.7-py2.7-linux-x86_64.egg/ZODB/__init__.py", line 17, in <module>
    from persistent import TimeStamp
  File "/usr/local/plone-4.3/primary/eggs/ZODB3-3.10.7-py2.7-linux-x86_64.egg/persistent/__init__.py", line 29, in <module>
    from zope.interface import classImplements
  File "/usr/local/plone-4.3/primary/eggs/zope.app.component-3.9.3-py2.7.egg/zope/__init__.py", line 3, in <module>
    import pkg_resources
  File "/usr/local/plone-4.3/primary/eggs/setuptools-33.1.1-py2.7.egg/pkg_resources/__init__.py", line 3019, in <module>
    @_call_aside
  File "/usr/local/plone-4.3/primary/eggs/setuptools-33.1.1-py2.7.egg/pkg_resources/__init__.py", line 3003, in _call_aside
    f(*args, **kwargs)
  File "/usr/local/plone-4.3/primary/eggs/setuptools-33.1.1-py2.7.egg/pkg_resources/__init__.py", line 3047, in _initialize_master_working_set
    for dist in working_set
  File "/usr/local/plone-4.3/primary/eggs/setuptools-33.1.1-py2.7.egg/pkg_resources/__init__.py", line 3047, in <genexpr>
    for dist in working_set
  File "/usr/local/plone-4.3/primary/eggs/setuptools-33.1.1-py2.7.egg/pkg_resources/__init__.py", line 2575, in activate
    for pkg in self._get_metadata('namespace_packages.txt'):
  File "/usr/local/plone-4.3/primary/eggs/setuptools-33.1.1-py2.7.egg/pkg_resources/__init__.py", line 2565, in _get_metadata
    for line in self.get_metadata_lines(name):
  File "/usr/local/plone-4.3/primary/eggs/setuptools-33.1.1-py2.7.egg/pkg_resources/__init__.py", line 1467, in get_metadata_lines
    return yield_lines(self.get_metadata(name))
  File "/usr/local/plone-4.3/primary/eggs/setuptools-33.1.1-py2.7.egg/pkg_resources/__init__.py", line 1463, in get_metadata
    value = self._get(self._fn(self.egg_info, name))
  File "/usr/local/plone-4.3/primary/eggs/setuptools-33.1.1-py2.7.egg/pkg_resources/__init__.py", line 1572, in _get
    with open(path, 'rb') as stream:
IOError: [Errno 13] Permission denied: '/usr/local/plone-4.3/primary/eggs/Products.PressRoom-3.18-py2.7.egg/EGG-INFO/namespace_packages.txt'
ERROR: Repozo command failed. See message above.
ERROR: Halting execution due to error.
```

**Comment by paregorios** [comment]
I saw the above error in the logs again this morning. Also a new one:

```
Traceback (most recent call last):
  File "bin/zeopack", line 351, in <module>
    import plone.recipe.zeoserver.pack
  File "/usr/local/plone-4.3/primary/eggs/plone.app.upgrade-1.4.5-py2.7.egg/plone/__init__.py", line 3, in <module>
    __import__('pkg_resources').declare_namespace(__name__)
  File "/usr/local/plone-4.3/primary/eggs/setuptools-33.1.1-py2.7.egg/pkg_resources/__init__.py", line 3019, in <module>
    @_call_aside
  File "/usr/local/plone-4.3/primary/eggs/setuptools-33.1.1-py2.7.egg/pkg_resources/__init__.py", line 3003, in _call_aside
    f(*args, **kwargs)
  File "/usr/local/plone-4.3/primary/eggs/setuptools-33.1.1-py2.7.egg/pkg_resources/__init__.py", line 3047, in _initialize_master_working_set
    for dist in working_set
  File "/usr/local/plone-4.3/primary/eggs/setuptools-33.1.1-py2.7.egg/pkg_resources/__init__.py", line 3047, in <genexpr>
    for dist in working_set
  File "/usr/local/plone-4.3/primary/eggs/setuptools-33.1.1-py2.7.egg/pkg_resources/__init__.py", line 2575, in activate
    for pkg in self._get_metadata('namespace_packages.txt'):
  File "/usr/local/plone-4.3/primary/eggs/setuptools-33.1.1-py2.7.egg/pkg_resources/__init__.py", line 2565, in _get_metadata
    for line in self.get_metadata_lines(name):
  File "/usr/local/plone-4.3/primary/eggs/setuptools-33.1.1-py2.7.egg/pkg_resources/__init__.py", line 1467, in get_metadata_lines
    return yield_lines(self.get_metadata(name))
  File "/usr/local/plone-4.3/primary/eggs/setuptools-33.1.1-py2.7.egg/pkg_resources/__init__.py", line 1463, in get_metadata
    value = self._get(self._fn(self.egg_info, name))
  File "/usr/local/plone-4.3/primary/eggs/setuptools-33.1.1-py2.7.egg/pkg_resources/__init__.py", line 1572, in _get
    with open(path, 'rb') as stream:
IOError: [Errno 13] Permission denied: '/usr/local/plone-4.3/primary/eggs/Products.PressRoom-3.18-py2.7.egg/EGG-INFO/namespace_packages.txt'
```

**Comment by alecpm** [comment]
I removed `eggs/Products.PressRoom-3.18-py2.7.egg` and re-ran buildout, which did not reinstall the egg. I also noticed that there were two sets of backup/pack cron jobs: one for the `plone_buildout` user and one for the `plone_daemon` user which we no longer use. I removed the duplicate cron tasks which likely triggered the permission/access issues, in addition to removing the unused egg which was inaccessible to the `plone_daemon` user.
